vllm/model_executor/model_loader/loader.py

```diff
@@ -307,7 +308,15 @@ def _get_weights_iterator(
                 hf_weights_files,
             )
         elif use_safetensors:
-            weights_iterator = safetensors_weights_iterator(hf_weights_files)
+            use_fastsafe_tensor = os.getenv('USE_FASTSAFETENSOR',
```

Comment:
My suggestion here would be to enable loader extra config for the DefaultLoader: https://github.com/vllm-project/vllm/blob/e784c6b9984e8f8116f74000b863d941495acb0b/vllm/model_executor/model_loader/loader.py#L190

So that you can enable fastsafetensors and GDS with
```  --model-loader-extra-config='{"use_fastsafetensors": true, "enable_gds: true"}'```